Code/GraphMol/SmilesParse/catch_tests.cpp

```diff
@@ -443,3 +443,20 @@ TEST_CASE("github#2450: getAtomSmarts() fails for free atoms", "[bug]") {
     CHECK(smiles == ":");
   }
 }
+
+TEST_CASE("MolFragmentToSmarts", "[Smarts]")
+{
+  SECTION("BasicFragment") {
+    auto m = "CCCCCN"_smiles;
+    std::vector<int> indices = {3, 4, 5};
+    auto smarts = MolFragmentToSmarts(*m, indices);
+    CHECK(smarts == "[#6]-[#6]-[#7]");
+  }
+  SECTION("FragmentInAromaticRing") {
```

Comment:
I am sure that it works, but it would still be good to also have a test on a mol constructed from SMARTS instead of SMILES.